### PR TITLE
Validation 

### PR DESCRIPTION
**Pull request by stevehotcodes**
I added validation for the sign up and sign in

### CODE REVIEW

File: src/app/components/signin/signin.component.html [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+        <div *ngIf="signinForm.get('email')?.hasError('required')" [style.color]="textColor" [style.font-size.rem]="errorMsgFontSize">email is required</div>
+    </div>
     <label>Password</label><br><br>
     <input type="password" formControlName="password"><br><br>
+    <div *ngIf="signinForm.get('password')!.errors && signinForm.get('password')!.touched">
+        <div *ngIf="signinForm.get('password')?.hasError('required')" [style.color]="textColor" [style.font-size.rem]="errorMsgFontSize">password is required</div>
+    </div> 
     
-    <button type="submit">Sign in</button>
+    <button type="submit" (click)="onSubmit()">Sign in</button>
```
Comment (JoshuaOndieki) [inline src/app/components/signin/signin.component.html]: with the onsubmit on the form, you don't need the click event on the button as long as it's type submit

Comment (stevehotcodes) [reply src/app/components/signin/signin.component.html]: no problem just fixing the problem

**Review by JoshuaOndieki — APPROVED** [review]

**Review by stevehotcodes — COMMENTED** [review]
okay

**Review by stevehotcodes — COMMENTED** [review]
ok

**Pull request merged**